### ISSUE TITLE
Issue with MultiSplitter in RTL mode

### ISSUE THREAD

**Issue by wisamidris7** [microsoft/fluentui-blazor#2351]
BarSize is not working with MultiSplitter

For example

```razor
<FluentMultiSplitter BarSize="20px" Height="150px" Style="border: 1px dashed var(--accent-fill-rest);">
    <FluentMultiSplitterPane Size="20%" Min="50px" Max="70%">
        Left Menu
    </FluentMultiSplitterPane>
    <FluentMultiSplitterPane Size="50%">
        <FluentMultiSplitter OnResize="@OnResizeHandler" Orientation="Orientation.Vertical">
            <FluentMultiSplitterPane Collapsible="true">
                Main Content
            </FluentMultiSplitterPane>
            <FluentMultiSplitterPane Collapsible="true">
                Console log
            </FluentMultiSplitterPane>
        </FluentMultiSplitter>
    </FluentMultiSplitterPane>
    <FluentMultiSplitterPane Size="30%">
        Properties
    </FluentMultiSplitterPane>
</FluentMultiSplitter>
```

It stays in 8

**Comment by vnbaaij** [comment]
It is working for me both on WebAssembly and Server version of the demo site:

```
<FluentMultiSplitter BarSize="15px" OnResize="@OnResizeHandler" Height="150px" Style="border: 1px dashed var(--accent-fill-rest);">
    <FluentMultiSplitterPane Size="20%" Min="50px" Max="70%">
        Left Menu
    </FluentMultiSplitterPane>
    <FluentMultiSplitterPane Size="50%">
        <FluentMultiSplitter  BarSize="25px" OnResize="@OnResizeHandler" OnExpand="@OnCollapseExpand" OnCollapse="@OnCollapseExpand" Orientation="Orientation.Vertical">
            <FluentMultiSplitterPane Collapsible="true">
                Main Content
            </FluentMultiSplitterPane>
            <FluentMultiSplitterPane Collapsible="true">
                Console log
            </FluentMultiSplitterPane>
        </FluentMultiSplitter>
    </FluentMultiSplitterPane>
    <FluentMultiSplitterPane Size="30%">
        Properties
    </FluentMultiSplitterPane>
</FluentMultiSplitter>
```

Gives me:
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/bfb2c636-966b-4408-a06f-00db1ec7f43a)

Can you share some more details about environment you are running on and versions you are using?

**Comment by wisamidris7** [comment]
Thanks it seems this problem from my project cause it worked in demo site
But the main issue is rtl


https://github.com/microsoft/fluentui-blazor/assets/104096256/579da33b-40b4-47ce-b7e1-3f43437bcf04

This issue only in MultiSplitter but in the Splitter component is fine

**Comment by dvoituron** [comment]
Fixed via #2380. Will be included in the next Release.
